Update cohorts docs (#6022)
PostHog/posthog.com: contents/docs/data/cohorts.mdx

            enterprise: true
---

Cohorts enable you to easily create a list of users who have something in common. You can use the cohorts to view a list of users who all did the same action or have the same property.

> Here are a few examples of the cohorts you can create:

Author: It's already mentioned above that a cohort is a list of users, so I dont think we need to say it again

> - Users who work at the same company.
> - Users who used your app in the last week.
> - Users who have churned from your product.

Reviewer: ```suggestion
- work at the same company.
- used your app in the last week.
- churned from your product.
- signed up recently.
- viewed the signup page, but didn't convert.
```

Author: Hmm I think it's clearer to have "Users who", but I like your additional examples. I'll add them

You can also analyze [trends](docs/product-analytics/trends), [insights](docs/product-analytics/insights), and [dashboards](/docs/product-analytics/dashboards) for your cohorts. This enables you to understand things like:

> - "Are people at company X using our app frequently?"
> - "What percentage of the users who signed up last week are still using our app?"

Reviewer: ```suggestion
- "Are people at company X using our app frequently?"
- "What percentage of the users who signed up last week are still using our app?"
- "What is the retention of our power users, and what features are they using?"
```

Author: Nice example

> - "Are people at company X using our app frequently?"
> - "What percentage of the users who signed up last week are still using our app?"

Author: ```suggestion
- "Are people at company X using our app frequently?"
- "What percentage of the users who signed up last week are still using our app?"
- "Which features are our power users using?"
```

## Cohorts vs. groups

Cohorts are often confused with [groups](/docs/data/groups), but they each serve different purposes:

- Cohorts represent a specific set of users – e.g., a list of users that all belong to the same company.

> - Groups aggregate events based on entities, such as organizations or companies, and do not necessarily have to be connected to a user. They enable you to analyze trends, insights, and dashboards at an entity-level, as opposed to a user-level.

Reviewer: ```suggestion
- Groups aggregate events based on entities, such as organizations or companies, and do not necessarily connect to a user. They enable you to analyze trends, insights, and dashboards at an entity-level, as opposed to a user-level.
```

 
Groups require additional code in your app to set up, while cohorts are created in PostHog and don't require additional code. This makes cohorts easier to use and quicker to get started. If your only goal is to create a list of **users** with something in common, we recommend cohorts instead of groups.

## How to create a cohort

There are two methods to create a new cohort:

### Method 1: Create a new cohort directly from an insight

> You can create cohorts using data from insights such as trends, [funnels](/product-analytics/funnels), [user paths](/docs/product-analytics/paths) and [lifecycle](/docs/product-analytics/lifecycle).

Reviewer: ```suggestion
You can create cohorts using data from insights such as trends, [funnels](/product-analytics/funnels), [user paths](/docs/product-analytics/paths) and [lifecycles](/docs/product-analytics/lifecycle).
```

> **Step 1:** Go to [insights](https://app.posthog.com/insights) and open any insight. Click on any data point to view the [persons](/docs/data/persons) represented in the underlying data.

Reviewer: ```suggestion
**Step 1:** Go to [insights](https://app.posthog.com/insights) and open any insight. Click a data point on the visualization to view the [persons](/docs/data/persons) represented in the underlying data.
```

**Step 2:** In the modal popup, click 'Save as cohort' in the bottom right.

The video below shows how to create a new cohort from a trend, although you can similarly create a cohort from a funnel, user path, or any other insight.

![How to create a cohort from a trend](../../images/features/cohorts/create-cohort-from-trend.mp4)

### Method 2: Create a new cohort from the cohorts page

**Step 1:** Navigate to the [Cohorts](https://app.posthog.com/cohorts) page in the sidebar:

**Step 2:** Click '+ New Cohort' on the top right.

![left hand navigation for cohorts](../../images/features/cohorts/cohorts-page.png)

> 

Reviewer: Set your filters for the cohort and press save?

Author: Not necessarily, since you either need to upload a csv, or set your filters. THis is explained in more detail below

I think this is sufficient here

## Static vs. dynamic cohorts

When creating a cohort, you'll choose between a static or dynamic cohort:

![two types of cohorts: static and dynamic](../../images/features/cohorts/static-and-dynamic-cohorts.png)

**Static cohorts** are lists of users that do not change over time. For example, all users who signed up for your app in January 2023. 

> Static cohorts are created by uploading a csv of users, or from insights. You can then update the list manually if you'd like to add or remove users.

Reviewer: How does someone update the list?

Reviewer: What is the format of the static list? What does it use to represent users?

> Static cohorts are created by uploading a csv of users, or from insights. You can then update the list manually if you'd like to add or remove users.

Reviewer: ```suggestion
Static cohorts are created by uploading a CSV file of users, or from insights. You can then update the list manually if you'd like to add or remove users.
```

> On the other hand, **dynamic cohorts** are defined by a continuously evaluated condition. This means that the members of the cohort can change over time. For example, a dynamic cohort could be all users who have have used a certain feature in your app.

Author: I dont think its needed, and I think may add confusion (need to be clearer that "last week" is continuously updated
